JPS_Ontology/KBTemplates/TBox/OntoSewage.csv

```diff
@@ -0,0 +1,169 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label

```

Comment:
I always thought we place both ths `.csv` and `.owl` file into the same folder under `JPS_Ontology/ontology/...`
@sm453 Am I mistaken?

Reply:
You are correct - both the `csv` and the `owl` file should be in the same subfolder of `JPS_Ontology/ontology/`. Unfortunately, many people have done this wrong, so it is easy to follow the numerous bad examples...

Reply:
- I moved csv file to the same directory as owl file.
- `ChannelType` sub-classes are added. We only added sub-classes which have more than 1% instances.
- The Miro board is now updated.
- Some sub-classes have same English translations. So, for these cases, we changed the translation. For example, `Manhole` sub-calss changed to `ManholeEndpoint`and `ManholeNodeType` for `Endpoint` and `NodeType` classes respectively.
- We are also not sure about the meaning of connection ID.

Reply:
> * Some sub-classes have same English translations. So, for these cases, we changed the translation. For example, `Manhole` sub-calss changed to `ManholeEndpoint`and `ManholeNodeType` for `Endpoint` and `NodeType` classes respectively.

I am not sure if that is a very useful approach (but maybe it is). I think the terms `Type` and `Node` add unnecessary confusion here:
![image](https://user-images.githubusercontent.com/79832739/205050372-ab2c2821-83b4-4310-a997-86dc869026fc.png)